extensions/xcode/package.json

```diff
@@ -72,7 +75,18 @@
       "title": "Create Swift Playground",
       "subtitle": "Xcode",
       "description": "Create a new Swift Playground and open it directly in Xcode",
-      "mode": "view"
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "playgroundDefaultLocation",
+          "type": "textfield",
+          "required": true,
+          "placeholder": "~/Desktop",
+          "default": "~/Desktop",
+          "title": "Default location",
+          "description": "Set the default location in which you want to save your Playgrounds. Ensure to provide an existing path or you will get an error on create."
```

Comment:
I guess the last sentence can be omitted `Ensure to provide an existing path or you will get an error on create.` as errors are gracefully handled by the `Create Swift Playground` command via a failure `Toast`.